### ISSUE TITLE
The predict_proba method does not exist in svm.SVC of sklearn in latest version 0.21.2

### ISSUE THREAD

**Issue by ZHHJemotion** [NUAA-AL/ALiPy#19]
Hi @NUAA-AL @tangypnuaa 

At first, thank you for your great work in Active Learning.

Today, when I implemented sklearn.svm.SVC in my active learning program based on uncertainly Sampling of Alipy, I found this error: "Exception: model object must implement predict_proba methods in current algorithm." I searched the related information of SVC on sklearn and I did find this predict_proba method in the latest version 0.21.2.  BUT why this error???

**Comment by tangypnuaa** [comment]
Hi, would you please post your code?

**Comment by ZHHJemotion** [comment]
> Hi, would you please post your code?

hi, @tangypnuaa,

my code as follow:
`  # SVC model
    model = svm.SVC(kernel='rbf', gamma=0.01, C=1)
    # The cost budget is 50 times querying
    stopping_criterion = alibox.get_stopping_criterion('num_of_queries', value=50)
    # US
    for round in range(round_count):
        # Get the data split of one fold experiment
        train_idx, test_idx, label_ind, unlab_ind = alibox.get_split(round)
        # Get intermediate results saver for one fold experiment
        saver = alibox.get_stateio(round)

        while not stopping_criterion.is_stop():
            # Select a subset of Uind according to the query strategy
            # Passing model=None to use the default model for evaluating the committees' disagreement
            select_ind = UncertaintyStrategy.select(label_ind, unlab_ind, model=model, batch_size=1)
            label_ind.update(select_ind)  # add select_ind into label_ind
            unlab_ind.difference_update(select_ind)  # remove select_ind from unlab_ind

            # Update model and calc performance according to the model you are using
            model.fit(X=X[label_ind.index, :], y=y[label_ind.index])
            pred = model.predict(X[test_idx, :])
            accuracy = alibox.calc_performance_metric(y_true=y[test_idx],
                                                      y_pred=pred,
                                                      performance_metric='accuracy_score')

            # Save intermediate results to file
            st = alibox.State(select_index=select_ind, performance=accuracy)
            saver.add_state(st)
            saver.save()

            # Passing the current progress to stopping criterion object
            stopping_criterion.update_information(saver)
        # Reset the progress in stopping criterion object
        stopping_criterion.reset()
        US_result.append(copy.deepcopy(saver))`
I modified from the example in the README.md. Thanks a lot.

**Comment by tangypnuaa** [comment]
Hi, it seems you do not enable probability estimates.
The default value of parameter `probability` is `False` in SVC.
Set it to True to solve this problem.
````
model = svm.SVC(kernel='rbf', gamma=0.01, C=1, probability=True)
````

**Comment by ZHHJemotion** [comment]
Hi, it works! Sorry that I pushed a so funny issue to you. Thank you so much!
